README.md

```diff
@@ -98,55 +98,52 @@ en0: flags=8863<UP,BROADCAST,SMART,RUNNING,SIMPLEX,MULTICAST> mtu 1500
 (The address we are seeking is the address following the `inet` field - in that particular case, `192.168.1.196`)
 
 Then you can start the container:
-#### Mac OS X
+#### Explicit port exposure (macOS compatible)
 ```bash
 docker run -p 50000-50050:50000-50050/udp -p 4000:4000/tcp -e INTEGRATED_TURN_PORT_RANGE=50000-50050 -e EXTERNAL_IP=<IPv4 address> -e VIRTUAL_HOST=localhost membrane_videoroom
 ```
-#### Ubuntu
+#### Using host network (Linux only)
 ```bash
 docker run --network=host -e EXTERNAL_IP=<IPv4 address> -e VIRTUAL_HOST=localhost membrane_videoroom
 ```
-> ***NOTE*** The first command, which is the one we suggest using on Mac OS, will also work completely on Ubuntu. At the same time, it is not necessarily true another way around, since there is a problem with running `--network=host` on `macOS`. That is why you need to explicitly forward the desired ports range on macOS. If you have decided to use the first command, make sure that the ports you are forwarding with the `-p` option switch of the `docker run` command are corresponding to the ports specified within the `INTEGRATED_TURN_PORT_RANGE` variable.
+> ***NOTE*** On macOS you need to explicitly forward ports since the `--network=host` option doesn't work there. Make sure that the ports you are forwarding with the option `-p` of the `docker run` command are corresponding to the ports specified within the `INTEGRATED_TURN_PORT_RANGE` variable. Please also keep in mind that the port range cannot be too wide as it might cause problems with the docker container starting.
```

Comment:
```suggestion 
> ***NOTE*** On macOS you need to explicitly publish ports since the `--network=host` option doesn't work there. Make sure that the ports you are publishing with the option `-p` of the `docker run` command are corresponding to the ports specified within the `INTEGRATED_TURN_PORT_RANGE` variable. Please also keep in mind that the port range cannot be too wide as it might cause problems with the docker container starting.
```

Reply:
done